pkg/argsbuilder/submit_tfjob.go

```diff
@@ -130,6 +133,8 @@ func (s *SubmitTFJobArgsBuilder) AddCommandFlags(command *cobra.Command) {
 	command.Flags().MarkDeprecated("cleanTaskPolicy", "please use --clean-task-policy instead")
 	command.Flags().StringVar(&s.args.CleanPodPolicy, "clean-task-policy", "Running", "How to clean tasks after Training is done, support Running, None and All.")
 
+	command.Flags().DurationVar(&activeDeadline, "active-deadline", activeDeadline, "Specifies the duration since startTime during which the job can remain active before it is terminated(e.g. '5s', '1m', '2h22m').")
```

Comment:
I suggest using the name `running-timeout` instead of `active-deadline`.

Reply:
Done